index.html

```diff
@@ -0,0 +1,43 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Fewlines Stylesheets demo</title>
+
+  <link rel="stylesheet" href="styles/fewlines.css">
+  
+  <style>
+    body { margin: 0; padding: 0; }
+
+    .open-font-test {
```

Comment:
so... if you did not use this class, nothing is in open-sans?

Reply:
Exactly yes.
The main reason is, i don't want to directly set something like 
```
h1 { font-family: var(--open-sans-font); }
```
Because, what if we don't want to use it? Like code sections on developpers-portal that use another font?
So this class is only here (and defined here) in order to test the import.

Reply:
But shouldn't we have both in the test file to see the difference?

Reply:
atm we just have this font. As soon as we add the second we should have both yes